examples/map/plot_frameless_image.py

```diff
@@ -38,6 +39,7 @@
 # sphinx_gallery_defer_figures
 
 ##############################################################################
-# At this point you could save the figure with ``plt.savefig()`` or show it:
+# At this point you could save the figure with :meth:`~matplotlib.pyplot.savefig`
+# or show it:
```

Comment:
Why? this should link to savefig no?

Reply:
DOes it not?

Reply:
it should have done before and wont now? I don't understand why you changed this?

Reply:
I feel like @Cadair is misreading...  It certainly wasn't linking before, and so this fix is in the right direction, but `:meth:` needs to be `:func:`

Reply:
Corrected.